cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -6503,11 +6595,10 @@ declare namespace Deno {
    */
   export function serve(
     options:
-      | ServeOptions
       | ServeTlsOptions
       | (ServeTlsOptions & TlsCertifiedKeyOptions),
-    handler: ServeHandler,
-  ): HttpServer;
+    handler: ServeTlsHandler,
```

Comment:
Do we need this specialized interface here? Feels to me like TLS should use `ServeHandler`

Reply:
We actually will need this eventually as we have to pass the TLS handshake data in. I can push this off for now but we definitely need it.